src/Stats.h

```diff
@@ -17,17 +18,27 @@ class Stats
 public:
     Stats();
     void reset(StatsEnum stat);
-    void increment(StatsEnum stat, int increase = 1, float floatIncrease = 1.0);
-    void incrementFloat(StatsEnum stat, float increase = 1.0);
+    void resetAll();
```

Comment:
Now that we have `resetAll`, why do we still need `reset`? Are there any situations where we only want to reset one of the stats fields?

Reply:
We reset the Current Run fields when we reenter the start tile. I could make this private though.